apps/docsite/src/components/BundleSizeTable.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
I think mdx format will work better here

Reply:
This section is supposed to contain only React code which we intend to use in our docsite... We can keep this isolated from Markdown stuff...
In the Markdown file we can simply import this component and use it

Reply:
I can switch this to tsx though